projects/netgrif-components-core/src/lib/data-fields/enumeration-field/enumeration-icon-field/abstract-enumeration-icon-field.component.ts

```diff
@@ -28,12 +39,12 @@ export abstract class AbstractEnumerationIconFieldComponent extends AbstractBase
         return this.dataField.component?.optionIcons.find(icon => icon.key === key)?.type;
     }
 
-    resolveArrow(key: string) {
-        return this.dataField.component?.properties.arrow === 'true';
+    resolveArrow() {
```

Comment:
Should be renamed, function does not resolve anything, now it is just getter

Reply:
done